Vwitter/Models/SpotifyTrack.m

```diff
@@ -0,0 +1,23 @@
+//
+//  SpotifyTrack.m
+//  Vwitter
+//
+//  Created by Christina Li on 7/28/22.
+//
+
+#import "SpotifyTrack.h"
+
+@implementation SpotifyTrack
+
+- (instancetype)initWithDictionary:(NSDictionary *)dictionary {
+    self = [super init];
+
+    if (self) {
+        self.uriString = dictionary[@"uri"];
+        self.trackName = dictionary[@"name"];
+        self.trackId = dictionary[@"id"];
```

Comment:
Shouldn't we initialize `endTimestamp` and `startTimestamp` here?

Also, this is very dangerous. You should be using your `CAST_TO_CLASS_OR_NIL` macro here for safety, and log an error if any of the fields are `nil`